src/content/docs/guides/integrating-with-fly-postgres.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Integrating a project with fly postgres
```

Comment:
```suggestion
title: Integrate A Project with Fly Postgres Database
```

Let's try this title. For the other ones, please use a similar letter casing (similar to what's on Notion)